firebase-web/src/test/java/io/spine/web/firebase/FirebaseDatabasePathTest.java

```diff
@@ -74,28 +77,30 @@ void testConstruct() {
     @Test
     @DisplayName("be tenant-aware")
     void testTenantAware() {
-        final TenantId domainTenant = TenantId.newBuilder()
-                                              .setDomain(InternetDomain.newBuilder()
-                                                                       .setValue("spine.io"))
-                                              .build();
-        final TenantId emailTenant = TenantId.newBuilder()
-                                             .setEmail(EmailAddress.newBuilder()
-                                                                   .setValue("john@doe.org"))
-                                             .build();
-        final TenantId firstValueTenant = TenantId.newBuilder()
-                                                  .setValue("first tenant")
-                                                  .build();
-        final TenantId secondValueTenant = TenantId.newBuilder()
-                                                   .setValue("second tenant")
-                                                   .build();
-        final List<String> paths = Stream.of(domainTenant,
-                                             emailTenant,
-                                             firstValueTenant,
-                                             secondValueTenant)
-                                         .map(FirebaseDatabasePathTest::tenantAwareQuery)
-                                         .map(FirebaseDatabasePath::allocateForQuery)
-                                         .map(FirebaseDatabasePath::toString)
-                                         .collect(toList());
+        InternetDomain domain = InternetDomainVBuilder.newBuilder()
+                                                      .setValue("spine.io")
+                                                      .build();
+        TenantId domainTenant = TenantIdVBuilder.newBuilder()
+                                                .setDomain(domain)
+                                                .build();
+        EmailAddress email = EmailAddressVBuilder.newBuilder()
+                                                 .setValue("john@doe.org")
+                                                 .build();
+        TenantId emailTenant = TenantIdVBuilder.newBuilder()
+                                               .setEmail(email)
+                                               .build();
+        TenantId firstValueTenant = TenantIdVBuilder.newBuilder()
+                                                    .setValue("first tenant")
+                                                    .build();
+        TenantId secondValueTenant = TenantIdVBuilder.newBuilder()
+                                                     .setValue("second tenant")
+                                                     .build();
+        List<String> paths = Stream.of(domainTenant, emailTenant, firstValueTenant,
```

Comment:
This does not read well right now. Could you put each of the parameters on a new line like it was before.